microprofile/testing/junit5/src/main/java/module-info.java

```diff
@@ -19,14 +19,18 @@
  */
 module io.helidon.microprofile.testing.junit5 {
 
-    requires io.helidon.microprofile.cdi;
```

Comment:
This must stay in - as `microprofile.server` is static, you made `microprofile.cdi` static as well, but it is always required.

Reply:
Fixed